public/js/common.js

```diff
@@ -11,6 +11,13 @@ const colours = Object.freeze({
     redLight      : 'rgba(243, 13, 20, 0.5)'
 });
 
+const snack = Object.freeze({
+    success     :   '<i class="material-icons">check</i>' + '&nbsp&nbsp&nbsp',
+    warning     :   '<i class="material-icons">warning</i>' + '&nbsp&nbsp&nbsp',
+    fail        :   '<i class="material-icons">block</i>' + '&nbsp&nbsp&nbsp',
+    close       :   '&nbsp&nbsp&nbsp' + '<i id=closeSnack class="material-icons">close</i>'
```

Comment:
why the plus? why not just have each as one string instead of addition of 2 strings?

Reply:
cause im silly :)

Reply:
fixed.